congame-doc/introduction.scrbl

```diff
@@ -1,37 +1,530 @@
+@section{Creating a new file}
 
-While the same person may of course run both servers and hence merge the data, in principle this provides a solution to this particularly aspect of privacy.
+Assuming you've  @seclink["Installing_Conscript_and_Racket"]{installed Racket and Conscript}, launch
+the DrRacket application. Start a new file. Click into the top/main area (the "definitions" window)
+and change the top line to:
 
-Since in many studies it is crucial to send messages to participants, the identities created by the identity server are emails that get received by the identity server. When it receives a message for a participant, it sends a notificaiton email to the participant that they received a message, which they can see by logging in. In order for this to work, one needs to run an SMTP proxy which deals with receiving emails from congame servers.
+@codeblock{
+#lang conscript/local
+}
 
-@section{How to configure Identity Server with SMTP Proxy}
+The first line of every Conscript @tech{study} program starts with @code{#lang conscript} or
+@code{#lang conscript/local}. The latter allows us to test drive our studies in the web browser,
+without setting up a server or databases. When you’re ready to start using it "for real", you change
+the first line to @code{#lang conscript} and then upload it to a Congame server.
 
-Currently the process is clunky and requires direct communication (as opposed to some admin functionality) between an admin of the congame server and an admin of the identity server. First, a new api user has to be created on the congame server, which will have an api_key. Then this api key should be used to insert the following information in the identity server's database:
+@;===============================================
 
-@code{"insert into congame_servers(name, url, key) values ('Dev Congame', 'http://localhost:5100', '<the-api-key>');"}
+@section{Writing the simplest study}
+
+Add some lines to your new program, so it looks like this:
+
+@codeblock|{
+#lang conscript/local
+
+(defstep (start)
+  @md{
+    # The Beginning is the End
+
+    This is all there is.
+  })
+
+(defstudy tutorial
+  [start --> start])
+}|
+
+This code defines a @tech{step} named @racket[start], and a @tech{study} named @racket[tutorial],
+which starts with a single step and ends with a single step.
+
+We'll get into the specifics later; but at a high level:
+
+@itemlist[
+
+@item{Conscript @tech{studies} are Racket programs that start @code{#lang conscript} or @code{#lang
+conscript/local}.}
+
+@item{In Conscript, you can use “@"@" notation” to intermingle code and text. This is explained
+further in @secref["Scribble_Syntax"].}
```

Comment:
The secref is not working on the snapshot you provided, it shows a red link.

Reply:
Got it. I retitled that section at the last minute, will fix.